Edds390/solarnetwork-aggregated-dashboard
# Feature/homepage aesthetics

Summary: Added minimalistic aesthetics to the homepage. The colour scheme is based on 6 colours, and images are done in photoshop. Most changes are done in css, with some changes to the js files of each respective page.

Changes:
1. Added a solarnode logo, centered and styled with css. Image is inserted by adding a img tag in the TopNavigationBar.js file. 
2. Added a background image. Image is inserted into the HomePage.js file with a image tag. Styled in the HomePage.css file. 
3. Changed the background colour of both the top navigation bar and the homepage to match.
4. Added 2 sample links of both sides of the logo. Implemented by adding a <p> tag and styled, positioned with css.
5. Added a custom font for the links at the homepage. The font is located at the src folder.

Main changes : HomePage.js, Homepage.css, TopNavigationBar.js, TopNavigationBar.css

Known Issues:
1. Currently using a custom font which works on my build. But not working for @Edds390. 
Potential solution: The css uses url() to the find the font and it is currently set as "../../fonts/SFOrston.ttf". This link maybe different for other developers, hence the text is not using the desired font.
2. Hovering over the button used to change its colour but now it is not reactive. 
<img width="1440" alt="screen shot 2018-03-27 at 11 55 43 am" src="https://user-images.githubusercontent.com/21161968/37937405-048ae7ee-31b6-11e8-96b8-271052c20dfb.png">

To do:
1. Integrate other pages to the links and the go button

Testing:
1. Just pull the changes, the visualisation should change. 
2. Check if there is a fonts folder inside the src folder.
3. Check If there is a icons folder inside the src folder.
4. Check if there is a solarnodeicon.png file and background.png file.
